modules/microshift-kickstart-ostree-install.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-kickstart-ostree-install_{context}"]
+= Creating a Kickstart file for a {microshift-short} RHEL for Edge installation
+
+Use the Kickstart file provided with the {microshift-short} RPM to provision an RHEL for Edge-based virtual machine.
+
+.Prerequisites
+
+* You set up the {microshift-short} Kickstart file.
+* You set up the [Activation Keys](https://console.redhat.com/insights/connector/activation-keys) and organization ID to activate your Red Hat subscription.
+//what else?
+
+[IMPORTANT]
+====
+The subscription must include access to the `rhocp-4.x-for-rhel-9-$(uname -m)-rpms` and `fast-datapath-for-rhel-9-$(uname -m)-rpms` RPM repositories.
+====
+
+.Procedure
+
+. Add the following variables to create an OSTree Kickstart file:
+
+* The `OSTREE_SERVER_URL` variable contains an OSTree server URL that is passed to the `ostreesetup` Kickstart command.
+* The `OSTREE_COMMIT_REF` variable contains an OSTree commit reference that is installed from the server.
+* The `AUTH_CONFIG` contents are copied to `/etc/ostree/auth.json` at the pre-install stage to authenticate access to the `OSTREE_SERVER_URL` server. If no server authentication is required, skip this setting.
+//Q: Can a user manually set the z-stream version?
```

Comment:
No, this is determined by the ostree commit. Whatever is there, will be installed.